REUSE.toml

```diff
@@ -53,7 +53,7 @@ SPDX-License-Identifier = "MIT"
 
 # Source/data files that we can't comment in.
 [[annotations]]
-path = "testlist.json"
+path = "test-config.json"
```

Comment:
There is no reason to change this filename, even if you are adding config options to each test.